hardware_interface/test/interface_manager_test.cpp

```diff
@@ -84,6 +84,93 @@ TEST(InterfaceManagerTest, InterfaceRewriting)
   EXPECT_EQ(2, foo_iface_ptr->foo);
 }
 
+struct TestCombinationHandle {
+  TestCombinationHandle(const std::string& name) : name{name} {}
```

Comment:
Nit: brace on newline and prefer parens over braces for init

```suggestion
struct TestCombinationHandle
{
  TestCombinationHandle(const std::string& name) : name(name) {}
```

Reply:
Applied in rebased commits